src/stories/components/MainMenu/styles.scss

```diff
@@ -5,6 +5,8 @@
 
 .main-menu {
   @include list-clean;
+
+  margin-top: calc(#{$spacing-10} * 2);
```

Comment:
@denglish7 menu should be in a fixed position on desktop. Apply the styles here instead of `.navigation`.